src/email-sent.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const fs = require('fs');
+const amministrazioni = require('../public/assets/data/authorities.db.json');
+const nodemailer = require('nodemailer');
+const mustache = require('mustache');
+
+module.exports = function(request, reply){
+    const referente = request.payload.nomeReferente;
+    const ipa = request.payload.ipa;
+    const url = request.payload.url;
+    const pec = amministrazioni[ipa].pec;
+    const amministrazione = amministrazioni[ipa].description;
+
+    // Generate test SMTP service account from ethereal.email
+    // Only needed if you don't have a real mail account for testing
+    nodemailer.createTestAccount((err, account) => {
+        // create reusable transporter object using the default SMTP transport
+        const transporter = nodemailer.createTransport({
+            host: 'smtp.ethereal.email',
+            port: 587,
+            secure: false, // true for 465, false for other ports
```

Comment:
Questo dovrebbe essere sempre true per la PEC. In ogni caso sposterei queste informazioni in un file di configurazione, che dici?

Reply:
Si sono perfettamente d'accordo. Utilizzerei un file di configurazione sia per parametrizzare queste proprietà e sia per poter utilizzare il sistema sia in Dev che il Prod. Se per te va bene, farei questo passaggio in una prossima PR.